sdk/web-pubsub/web-pubsub/swagger/README.md

```diff
@@ -12,17 +12,64 @@ generate-metadata: false
 license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../
 source-code-folder-path: ./src/generated
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/webpubsub/data-plane/WebPubSub/preview/2021-05-01-preview/webpubsub.json
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/webpubsub/data-plane/WebPubSub/preview/2021-08-01-preview/webpubsub.json
 add-credentials: false
-package-version: 1.0.0-beta.3
+package-version: 1.0.0-beta.4
 v3: true
 hide-clients: true
 use-core-v2: true
-use-extension:
-  "@autorest/modelerfour": "4.19.3"
 ```
 
 ## Customizations for Track 2 Generator
 
 See the [AutoRest samples](https://github.com/Azure/autorest/tree/master/Samples/3b-custom-transformations)
 for more about how we're customizing things.
+
+### Rename maxresults -> maxPageSize
+
+```yaml
+directive:
+  - from: swagger-document
+    where: $["paths"]["/api/hubs/{hub}/:generateToken"]["post"]["parameters"]
```

Comment:
ah a shame you have to do so much work to find the positional parameter, I think this is why storage's swagger uses ref's to parameters defined in a central place: 

https://raw.githubusercontent.com/Azure/azure-rest-api-specs/4a93ab078fba7f087116283c8ed169f9b8e30397/specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-10-02/blob.json 

This permits much easier transforms:

https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/storage/storage-blob/swagger/README.md#rename-timeoutparameter---timeoutinseconds

Reply:
Yeah, alas. I also suspect there is an easier way to use the selectors to find the right item so you don't need the loop, but I saw the loop being used elsewhere and I couldn't figure out the query syntax, so here we are!